### PR TITLE
Frontend app cleanup

### PR DESCRIPTION
**Pull request by ssalka**
Just a quick bit of cleanup on the existing frontend code. Not 100% committed to the things done here, just doing a couple small things that felt obvious to me

* Normalized dependencies from both workspaces into top-level `yarn.lock` (this is [apparently](https://classic.yarnpkg.com/en/docs/workspaces/#toc-why-would-you-want-to-do-this) how yarn workspaces work)
* Added `pages` folder to match URL structure
* Fixed duplicate `hello` message being rendered

### CODE REVIEW

File: packages/unigraph-dev-explorer/src/App.tsx [hunk 1]
```diff
@@ -1,112 +1,73 @@
...
+function App() {
   const classes = useStyles();
 
   return (
     <div className={classes.root}>
       <CssBaseline />
       <Router>
         <AppBar position="fixed" className={classes.appBar}>
-          <Toolbar>
+          <Toolbar variant="dense">
```
Comment (ssalka) [inline packages/unigraph-dev-explorer/src/App.tsx:48]: doesn't have to be dense, just thought it looked slightly better at the time

File: packages/unigraph-dev-explorer/src/App.tsx [hunk 2]
```diff
@@ -1,112 +1,73 @@
...
-        <ListItemIcon><Info/></ListItemIcon>
-        <ListItemText primary="About" />
-      </ListItem>
-    </List>
-  </Drawer>
-}
+// REVIEW: use react state? why is window.unigraph needed?
+// must keep single reference to this, otherwise react
+// mounting the app causes a 2nd connection to be created
+window.unigraph = unigraph("ws://localhost:3001");
```
Comment (ssalka) [inline packages/unigraph-dev-explorer/src/App.tsx:38]: here I'm confused about why this global var is needed – should maybe sync on this

Comment (HaojiXu) [reply packages/unigraph-dev-explorer/src/App.tsx:39]: I'm using window globals to keep it framework-agonistic: at least from my perspective, if anyone want to create applications for unigraph, they should feel free to choose from React, Angular, jQuery, or even just vanilla JS. 

Perhaps UMD global is a better way of approaching this? I'm not quite familiar with module management and we can definitely talk more about this!

**Review by HaojiXu — APPROVED** [review]
Looks good to merge!

**Pull request merged**